chain/network/src/peer_manager/testonly.rs

```diff
@@ -0,0 +1,57 @@
+use crate::network_protocol::testonly as data;
+use crate::peer_manager::peer_manager_actor::Event as PME;
+use crate::testonly::actix::ActixSystem;
+use crate::testonly::fake_client;
+use crate::PeerManagerActor;
+use crate::Sink;
+use actix::Actor;
+use near_network_primitives::types::NetworkConfig;
+use near_store::test_utils::create_test_store;
+use std::sync::Arc;
+use tokio::sync::mpsc;
+
+#[derive(Debug, PartialEq, Eq)]
+pub enum Event {
+    Client(fake_client::Event),
+    PeerManager(crate::peer_manager::peer_manager_actor::Event),
+}
+
+pub struct Handler {
```

Comment:
In this case `Handler` feels a bit contextless

Reply:
renamed to ActorHandler